sprints/20210129-planning-maintenance-self-serve/sprint-review.md

```diff
@@ -4,13 +4,15 @@ At the beginning of 2021, our team headcount decreased. With two previous iterat
 the third was aimed at planning, maintaining and improving our systems self-serve.
 
 # Details
+**Product Owner**:
 - geckoview/android-components cleanup: aki
 - automate mac signer maintenance: bhearsum
 - password rotations:
 - pushhealth: sclements
 - interruptduty:
 - greening-up tests: jmaher
 - releng 2021 planning:
+**Scrum Master**: unassigned
```

Comment:
You've done a lot of heavylifting here, @escapewindow should I put your name down instead?

Reply:
Possibly. I'm not really a real scrum master, maybe me+team?